android/app-example/src/main/java/com/badoo/ribs/example/rib/foo_bar/builder/FooBarModule.kt

```diff
@@ -42,18 +45,27 @@ internal object FooBarModule {
             permissionRequester = permissionRequester
         )
 
+    @FooBarScope
+    @Provides
+    @JvmStatic
+    internal fun viewPlugins(): Set<@JvmSuppressWildcards ViewPlugin> = setOf(
+        ParentLongClickListener()
+    )
+
     @FooBarScope
     @Provides
     @JvmStatic
     internal fun node(
         savedInstanceState: Bundle?,
         customisation: FooBar.Customisation,
         router: FooBarRouter,
-        interactor: FooBarInteractor
+        interactor: FooBarInteractor,
+        viewPlugins: Set<@JvmSuppressWildcards ViewPlugin>
```

Comment:
maybe this set can be dynamic (e.g. added using callback methods) ?
that way we can attach/detach listeners